docs/data/xrootd/install-cms-xcache.md

```diff
@@ -0,0 +1,242 @@
+Installing the CMS XCache
+===============================
+
+This document describes how to install a CMS XCache.  This service allows a site or regional
+network to cache data frequently used by the [CMS experiment](https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/CMS.yaml), reducing data transfer over the wide-area network and
+decreasing access latency.
+
```

Comment:
I'd still like a sentence here about the two different configurations: standalone + multi-node

Reply:
ok fixed